leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/LeshanServer.java

```diff
@@ -140,6 +154,46 @@ public LeshanServer(final Set<Endpoint> endpoints, final ClientRegistry clientRe
         createServer(endpoints);
     }
 
+    public LeshanServer(final ClientRegistry clientRegistry, final SecurityRegistry securityRegistry,
```

Comment:
Though the CoapEndpoint are highly depending on "internal" stuff, creating endpoints outside the LeshanServer is very complex (or impossible).